.github/repo-diff-msg.sh

```diff
@@ -6,16 +6,22 @@ echo "Bump of Kairos repositories" > pr-message
 echo "--------------------------" >> pr-message
 DIFF=$(diff -u build/versions.old.yaml build/versions.new.yaml)
 if [[ $? == 1 ]]; then
-  echo "> [\!WARNING]" >> pr-message
-  echo "> There were changes to installed packages" >> pr-message
-  echo "\`\`\`diff" >> pr-message
-  echo "${DIFF}" >> pr-message
-  echo "\`\`\`" >> pr-message
-  echo "\n" >> pr-message
+  {
+    echo "> [\!WARNING]"
+    echo "> There were changes to installed packages"
+    echo "\`\`\`diff"
+    echo "${DIFF}"
+    echo "\`\`\`"
+    printf '\\n'
```

Comment:
I'm not sure if we actually wanted a literal `\n` or a rendered new line. Given a simple `echo ""` would print a new line, I assumed we wanted a literal bashlash-n

Reply:
I think we want an enter actually because the literal \n doesn't help here: https://github.com/kairos-io/kairos/pull/1744#pullrequestreview-1587408185

Reply:
True! I wasn't sure where the the text was actually printed, thanks! Will change to a simple `echo` then.

Reply:
done